tests/Doctrine/Tests/ORM/Functional/Ticket/DDC933Test.php

```diff
@@ -29,7 +30,9 @@ public function testLockCTIClass()
         $this->_em->flush();
 
         $this->_em->beginTransaction();
-        $this->_em->lock($manager, \Doctrine\DBAL\LockMode::PESSIMISTIC_READ);
+        $this->_em->lock($manager, LockMode::PESSIMISTIC_READ);
         $this->_em->rollback();
+
+        $this->addToAssertionCount(1);
```

Comment:
I'm not sure, the issue mentioned errors and bad SQL, which would probably cause some exceptions (https://github.com/doctrine/doctrine2/issues/5462).